util/debug-db/debug-db.go

```diff
@@ -0,0 +1,110 @@
+package main
```

Comment:
This is a command line util that you can use to add an admin/admin user to bbolt datastores you've gotten from users, so that you can debug it without having to ask for a valid username/password

Reply:
should it have a cmd folder?

Reply:
Maybe? I could also put it in ziti. Wasn't really sure where the best place was to put a debugging tool like this. Certainly don't want to ship it with ziti-all :)